Formula/t/three-body.rb

```diff
@@ -0,0 +1,18 @@
+class ThreeBody < Formula
+  desc "三体编程语言 Three Body Language written in Rust"
+  homepage "https://github.com/rustq/3body-lang"
+  url "https://github.com/rustq/3body-lang/releases/download/0.3.0-beta/3body-lang.tar.gz"
```

Comment:
We don't ship beta software, see: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
Ok thanks, I will convert this pr to draft status to make it correct for a period time